run_service.sh

```diff
@@ -317,8 +317,8 @@ ask_password_if_needed() {
 
 # Validates the provided password
 validate_password() {
-    local is_password_valid_1=$(poetry run $PYTHON_CMD ../scripts/is_keys_json_password_valid.py ../$keys_json_path $password_argument)
-    local is_password_valid_2=$(poetry run $PYTHON_CMD ../scripts/is_keys_json_password_valid.py ../$operator_keys_file $password_argument)
+    local is_password_valid_1=$(poetry run python ../scripts/is_keys_json_password_valid.py ../$keys_json_path $password_argument)
+    local is_password_valid_2=$(poetry run python ../scripts/is_keys_json_password_valid.py ../$operator_keys_file $password_argument)
```

Comment:
Please explain.

Reply:
$PYTHON_CMD is the auto-detected python command on the user's system.

Within the virtual environment, all the commands were using just `python`, so these 2 commands were an exception which I fixed.